### PR TITLE
Update JSS dependency to ^6.0.0

### PR DESCRIPTION
**Pull request by EvanDarwin**
Update JSS dependency so this works with Node >7.0.0.

Related to Issue #58

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,6 @@
     "react-bootstrap": "^0.30.6",
     "react-dom": "^15.3.2",
     "react-input-enhancements": "^0.5.3",
-    "react-pure-render": "^1.0.2",
```
Comment (alexkuz) [inline package.json:54]: Let's not remove it until we use `PureComponent` :)

Comment (EvanDarwin) [reply package.json:54]: I removed it because it was a duplicate entry, it is still included in the `dependencies` section.

**Comment by 0rvar** [comment]
What's the status on this?

File: package.json (hunk 1 above)
Comment (alexkuz) [reply package.json:54]: Ah, my bad

**Comment by alexkuz** [comment]
Thank you!

@awestroke published in `v0.10.0`

**Pull request merged**